### PR TITLE
Upgrade REPL

### PR DESCRIPTION
**Pull request by developit**
Depends on #379 (retarget once merged).

### CODE REVIEW

File: src/components/code-block/index.js [hunk 1]
```diff
@@ -68,6 +72,8 @@ function HighlightedCodeBlock({ code, lang, ...props }) {
 		code.replace(/</g, '&lt;').replace(/>/g, '&gt;');
 	const htmlObj = useMemo(() => ({ __html: html }), [html]);
 
+	console.log({ canHighlight, pending, error });
```
Comment (marvinhagemeister) [inline src/components/code-block/index.js]: whoops :slightly_smiling_face:

File: src/components/controllers/repl/index.js [hunk 2]
```diff
@@ -108,6 +108,7 @@ export default class Repl extends Component {
 	}
 
 	render(_, { loading, code, error, example, copied }) {
+		console.log('error: ', error);
```
Comment (marvinhagemeister) [inline src/components/controllers/repl/index.js]: Let's only log when there is an error :+1:

File: src/components/controllers/repl/runner.js [hunk 3]
```diff
@@ -99,7 +104,7 @@ export default class Runner extends Component {
 		}
 		if (vnode) {
 			try {
-				this.root = render(vnode, this.base, this.root);
+				this.root = render(vnode, this.base);
```
Comment (marvinhagemeister) [inline src/components/controllers/repl/runner.js]: I wonder if we should wrap a component with a `componentDidCatch` lifecycle all around the users code.

Comment (developit) [reply src/components/controllers/repl/runner.js]: ooh I like that idea!

**Pull request merged**